build.gradle

```diff
@@ -20,5 +21,20 @@ buildscript {
 allprojects {
     repositories {
         jcenter()
+        maven { url "https://www.jitpack.io" }
     }
 }
+
+ext {
+    // Sdk and tools
+    minSdkVersion = 11
+    targetSdkVersion = 25
```

Comment:
Min is API 11 Honeycomb Android. Previously it was API 16 Jelly Bean Android.
We can move that to API 16 if required.